src/Paket.Core/Packaging/NupkgWriter.fs

```diff
@@ -83,23 +84,45 @@ module internal NupkgWriter =
             frameworkAssembliesList |> List.iter (buildFrameworkReferencesNode >> d.Add)
             metadataNode.Add d
 
-        let buildDependencyNode (Id, requirement:VersionRequirement) =
+        let buildDependencyNode (id, requirement:VersionRequirement) =
```

Comment:
Can you please rename this thing. id is a function from the FSharp.Core and I think we should avoid confusion. Maybe name or idProperty or something like that